src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1389,11 +1389,22 @@
   
     <Field Name="InitializerExpressionOpt" Type="BoundExpression" Null="allow"/>
   </Node>
-  
+
   <Node Name="BoundDelegateCreationExpression" Base="BoundExpression">
```

Comment:
No, there is an empty line before every node type in this file, including this one.